bcda/api/requests.go

```diff
@@ -155,6 +163,75 @@ func (h *Handler) buildBulkGroupRequest(r *http.Request) (service.RequestType, e
 	return reqType, nil
 }
 
+func (h *Handler) JobsStatus(w http.ResponseWriter, r *http.Request) {
+	var (
+		ad          auth.AuthData
+		statusTypes []models.JobStatus
+		err         error
+	)
+
+	statusTypes = models.AllJobStatuses // default request to retrieve jobs with all statuses
+	params, ok := r.URL.Query()["_status"]
+	if ok {
+		statusMap := make(map[string]struct{})
+		rawStatusTypes := strings.Split(params[0], ",")
+		statusTypes = nil
+
+		// validate no duplicate status types
+		for _, status := range rawStatusTypes {
+			if _, ok := statusMap[status]; !ok {
+				statusMap[status] = struct{}{}
+				statusTypes = append(statusTypes, models.JobStatus(status))
+			} else {
+				errMsg := fmt.Sprintf("Repeated status type %s", status)
+				h.RespWriter.Exception(w, http.StatusBadRequest, responseutils.RequestErr, errMsg)
```

Comment:
Perhaps we should look into making a ticket (for the H-team?) to see if it's better to dedup the list and just continue with the request?